test/unit/CMakeLists.txt

```diff
@@ -3,23 +3,21 @@ set(source
       strpak.unit.cpp
       xiopak.unit.cpp
       cvpak.unit.cpp
-      $<TARGET_OBJECTS:cse_libstubs>
+      ../../src/libstubs.cpp
```

Comment:
Use `${CSE_SOURCE_DIR}` instead of `../../src`.

Reply:
OK